### PR TITLE
UI automation changes for ci

### PR DESCRIPTION
**Pull request by shahzaibj**
Changes per changes made in the following:

- AzureAD/microsoft-authentication-library-common-for-android#1094
- AzureAD/microsoft-authentication-library-common-for-android#1127

### CODE REVIEW

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/AbstractMsalBrokerTest.java [hunk 1]
```diff
@@ -44,20 +42,14 @@
...
-    @Rule(order = 7)
-    public final TestRule powerLiftIncidentRule = new PowerLiftIncidentRule(mBroker);
-
-    @Rule(order = 8)
-    public final TestRule deviceEnrollmentFailureRecoveryRule = new DeviceEnrollmentFailureRecoveryRule();
-
+    @NonNull
     @Override
     public ITestBroker getBroker() {
+        // only initialize once....so calling getBroker from anywhere returns the same instance
```
Comment (iambmelt) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/AbstractMsalBrokerTest.java:48]: Are these parallelized? (Or could they be in the future?) Does this need to be a `synchronized` call?

**Review by iambmelt — APPROVED** [review]
Minor question otherwise LGTM thanks @shahzaibj !

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/AbstractMsalBrokerTest.java (hunk 1 above)
Comment (shahzaibj) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/AbstractMsalBrokerTest.java:48]: I can make it synchronized....but this is currently only called once per test case (each test case would create new instance of this class)

Comment (shahzaibj) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/AbstractMsalBrokerTest.java:48]: sorry that wasn't entirely correct

Comment (shahzaibj) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/AbstractMsalBrokerTest.java:48]: It is not parallelized right now......I can make it synchronized....but each test case would create new instance of this class...so it shouldn't really matter

**Review by rpdome — APPROVED** [review]

**Pull request merged**